src/main/java/com/jagrosh/jmusicbot/commands/music/PlaylistsCmd.java

```diff
@@ -34,11 +35,15 @@ public PlaylistsCmd(Bot bot)
         this.aliases = bot.getConfig().getAliases(this.name);
         this.guildOnly = true;
         this.beListening = false;
-        this.beListening = false;
     }
     
     @Override
     public void doCommand(CommandEvent event) 
+    {
+        this.replyList(event, true);
+    }
+
+    public void replyList(CommandEvent event, boolean showInstructions)
```

Comment:
Make this method static & update the arguments to include the playlist loader.
```suggestion
    public static void replyList(CommandEvent event, PlaylistLoader loader, boolean showInstructions)
```